components/bl-markdown-component/src/index.js

```diff
@@ -34,12 +36,17 @@ export default function Markdown({ component }) {
 
   const setContent = text => setMarkdown(md.render(text));
 
+  useEffect(() => {
+    component.el = markdownRef.current;
+  }, []);
```

Comment:
Should we add markdownRef.current to the deps?

Reply:
@StasivPavlo use `elRef` instead